README.md

```diff
@@ -30,14 +38,22 @@ DokuWiki is a simple to use and highly versatile Open Source wiki software that
 
 ### Supported architectures
 
-* x86-64b - ![](https://ci-apps.yunohost.org/ci/logs/dokuwiki%20%28Official%29.svg)
-* ARMv8-A - ![](https://ci-apps-arm.yunohost.org/ci/logs/dokuwiki%20%28Official%29.svg)
-* Jessie x86-64b - ![](https://ci-stretch.nohost.me/ci/logs/dokuwiki%20%28Official%29.svg)
+* x86-64b - [![Build Status](https://ci-apps.yunohost.org/ci/logs/dokuwiki%20%28Official%29.svg)](https://ci-apps.yunohost.org/ci/apps/dokuwiki/)
+* ARMv8-A - [![Build Status](https://ci-apps-arm.yunohost.org/ci/logs/dokuwiki%20%28Official%29.svg)](https://ci-apps-arm.yunohost.org/ci/apps/dokuwiki/)
+* Jessie x86-64b - [![Build Status](https://ci-stretch.nohost.me/ci/logs/dokuwiki%20%28Official%29.svg)](https://ci-stretch.nohost.me/ci/apps/dokuwiki/)
 
 ## Limitations
 
 * Cannot create or login with dokuwiki internal users, only users from Yunohost (Work needed for [authchained plugin](https://www.dokuwiki.org/plugin:authchained))
 
+## Additional information
+
+### Changelog
+
+* *Many missing - List taken from previous documentation*
```

Comment:
Probably yes.
But keeping a changelog up to date is boring...
We should find a way to handle that properly

Reply:
Well, the `CHANGELOG `file seems easier to me compared to putting stuff into the `README`. It seems to be used broadwide too. I will begin to work on this on my side
It should be discussed yes!

Reply:
Yep of course using a CHANGELOG file is the best way, that just that I'm worried about how to fill it.
If it would be manual, I don't think it will be up to date...

Reply:
The README is not updated too ;)
And you can only put "one" change per line. When I refactored the package, they were many things to write

Reply:
I was thinking maybe about a way to fill a changelog automatically.
I've written a helper that do that. Don't know tough if it's a best thing do to.

Anyway, I won't argue that having a changelog would be nice.